app/templates/base.njk

```diff
@@ -54,16 +54,14 @@
 </head>
 <body>
   <!-- Banner -->
-  <div class="bg-dark">
+  <!-- <div class="bg-dark">
     <div class="bg-info-gradient py-2">
       <div class="container text-center">
-        <a class="text-white" href="/$( _lang_ )$/bitcoin-cash-website-notice">
-          $( website-notice.banner.title )$<br>
-          $( website-notice.banner.sub )$
+        <a class="text-white" href="/$( _lang_ )$/">
         </a>
       </div>
     </div>
-  </div>
+</div> -->
```

Comment:
Leaving this here in case we need a banner in the future?

Reply:
Yes, that was the idea.